10326.xml

```diff
@@ -0,0 +1,724 @@
+        <RangeEnumeration/>
+        <Units>°C</Units>
+        <Description><![CDATA[Battery Temperature.]]></Description>
+      </Item>	  
+	  <Item ID="301">
+        <Name>Temperature</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>°C</Units>
+        <Description><![CDATA[Environment Temperature.]]></Description>
+      </Item>
+      <Item ID="302">
+        <Name>Humidity</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>0..100</RangeEnumeration>
+        <Units>%</Units>
+        <Description><![CDATA[Environment Humidity.]]></Description>
+      </Item>
+      <Item ID="303">
+        <Name>PM2.5</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>μg/m3</Units>
+        <Description><![CDATA[PM2.5 of the environment.]]></Description>
+      </Item>
+      <Item ID="304">
+        <Name>Smog</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>μg/m3</Units>
+        <Description><![CDATA[Smoke concentration of the environment.]]></Description>
+      </Item>
+      <Item ID="305">
+        <Name>CO</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>ppm</Units>
+        <Description><![CDATA[CO concentration of the environment.]]></Description>
+      </Item>
+      <Item ID="306">
+        <Name>CO2</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>ppm</Units>
+        <Description><![CDATA[CO2 concentration of the environment.]]></Description>
+      </Item>
+      <Item ID="307">
+        <Name>PM10</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>μg/m3</Units>
+        <Description><![CDATA[PM10 of the environment.]]></Description>
+      </Item>
+	  
+	  
+      <Item ID="601">
+        <Name>Speed</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>M/H</Units>
```

Comment:
Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

Reply:
Have commented